packages/blade/src/components/Collapsible/CollapsibleLink.tsx

```diff
@@ -45,9 +45,13 @@ const StyledCollapsibleLinkIcon = styled(BaseBox)<StyledCollapsibleLinkIconProps
 
 // Not really an IconComponent, a wrapper is needed for animating the icon inside
 const CollapsibleLinkIcon: IconComponent = (props) => {
-  const { isExpanded, direction } = useCollapsibleContext();
+  const { isExpanded, direction } = useCollapsible();
   return (
-    <StyledCollapsibleLinkIcon isExpanded={isExpanded} direction={direction}>
+    <StyledCollapsibleLinkIcon
+      isExpanded={isExpanded}
+      direction={direction}
+      {...makeAccessible({ hidden: true })}
```

Comment:
why hidden?

Reply:
It's a presentational only icon so making it hidden from screen readers